icons/greensock.svg

```diff
@@ -1 +1 @@
03.5543 1.1848.7307.1794.1333.1844.1403.1684.2435-.0371.2336.1123.4912.407.6967.0751.052.1222.1072.1222.1433 0 .0722.1965.1704.3438.1704.0802 0 .1454-.034.3138-.1664zm4.8263-1.317c.455.013.48-.0191.2355-.3018-.2546-.2957-.3238-.5332-.2305-.7958l.037-.1023L19.9047 9a1.1697 1.1697 0 00-.1634-.0721 7.9837 7.9837 0 00-.0792.3107c-.0581.2406-.1303.858-.1313 1.1226 0 .023.0381.013.1053-.028.0922-.0552.1403-.0622.3859-.0552zm-1.5216-.0251c.2736-.015.5573-.0361.6315-.0461l.1343-.018.017-.4531c.023-.6415.1153-1.0204.394-1.6258.1984-.431.2816-.809.2405-1.0886-.018-.1223-.025-.1303-.1884-.2014a4.305 4.305 0 00-.9362-.2536c-.3107-.0451-1.2169-.019-1.5587.045-.1403.0261-.3979.0803-.5713.1213-.427.1003-.6535.0873-.9783-.055-.3207-.1414-.5252-.2687-.7187-.4491-.1513-.1414-.1654-.1484-.2065-.1013-.03.0351-.047.1404-.057.3689-.0141.3478-.0662.5312-.2206.7788a.834.834 0 00-.0862.1604c0 .012.1193.1463.2646.2967.5002.5212.7458.6555 1.1407.6254.1303-.01.2205-.006.2205.01 0 .0291-.2546.2306-.3618.2867-.0482.0251-.0602.0532-.0522.1133l.0602.409c.036.2365.0641.3438.1002.3758.0932.0802.5784.3308.882.4531.402.1634.6185.2215.9262.2476.374.031.382.031.9242 0zm-3.2376-.428c.008-.015-.022-.2406-.0701-.5032-.0632-.3568-.0983-.4861-.1374-.5192-.03-.024-.1734-.1283-.3197-.2316a5.5159 5.5159 0 01-.5383-.453l-.2716-.2646-.1423.0751-.1544.0832a.7016.7016 0 00.0642.1765c.04.0932.1262.3237.1904.5111.1654.4902.2917.7247.6174 1.1507l.1243.1614.2306.0361c.1263.02.2576.0461.2917.0591.056.021.0631.012.0801-.1152a.7568.7568 0 01.0341-.1664zm4.9596-4.8714c.2005-.019.2346-.0501.1474-.1383-.0522-.0532-.1795-.0592-.2807-.012-.0682.031-.0822.024-.1874-.0903-.1103-.1223-.1163-.1253-.2546-.1082-.1253.015-.1444.01-.1604-.0411a1.0805 1.0805 0 00-.1804-.2266.844.844 0 00-.3258-.2305c-.2436-.0922-.3789-.0822-.6104.0451-.6686.3659-.6415.3619-1.286.1975-.6044-.1544-1.0565-.3077-1.1998-.408-.1203-.0832-.38-.1704-.5944-.2004-.1604-.0221-.2646.013-.417.1403l-.0802.0661.0722.1283c.1022.1815.0922.1915-.0622.0602-.1633-.1404-.3578-.2346-.486-.2356-.0893 0-.356.1153-.388.1684-.009.014.0732.033.1814.0431.1153.01.2717.0481.377.0932.2725.1163.4149.1544.6655.1805.2736.026.7728-.005.83-.0532.029-.025.1152-.029.2966-.015.2386.018.3378.04.9051.1935.3288.0902.5593.0831.9512-.0261.1805-.0501.407-.1002.5022-.1103.1604-.017.1925-.01.441.1033.401.1804.6115.2967.7298.401.1172.1001.1303.1032.414.076zm-2.2332-.6856l.2386-.1243-.005-.2456a11.992 11.992 0 00-.022-.4961l-.0171-.2516-.2386.1844c-.1413.1103-.2786.1925-.3398.2035-.1714.033-1.0184 0-1.0986-.0421a2.3154 2.3154 0 01-.2917-.2596l-.2205-.2205-.003.1303c0 .0701-.011.1924-.024.2686-.022.1383-.02.1423.1303.3338.0842.1072.1523.2024.1523.2125 0 .0761 1.1307.427 1.3853.431.0812 0 .1844-.0361.3538-.1243zm-.453-.8911c.2826-.0541.421-.2586.5242-.7748.1102-.5553.1664-1.4284.1132-1.8012-.047-.3348-.42-.4862-1.1386-.4621-.445.015-.6295.0591-.7809.1874-.0942.0802-.0962.0872-.1373.4581-.0802.7127.0361 1.728.2406 2.11.0972.1834.1844.2505.3658.2806.2035.035.6365.036.813.002z"/></svg>
```

Comment:
There seem to be some differences between PR (red) and source (black) - any chance you could regenerate from source? It's quite a complex logo unfortunately... 😅
![image](https://user-images.githubusercontent.com/1830867/103036592-08b31e00-4562-11eb-8ee0-7f45b485aece.png)

Reply:
@adamrusted Done. 🦸🏼‍♂️ 🧦  And yeah, it took me quite some time. 😄  I've removed the shaddows of the cape by the way. I think it looks better without and this way it also matches the already removed shaddows of the body.

![greensock](https://user-images.githubusercontent.com/70264417/103135196-f696c400-46b6-11eb-8bf2-dfe7c185bd42.png)